proto/xudrpc.proto

```diff
@@ -380,20 +385,24 @@ message SwapFailure {
 
 message GetInfoRequest {}
 message GetInfoResponse {
-  // The version of this instance of xud.
-  string version = 1 [json_name = "version"];
```

Comment:
Not a huge deal since we are still in pre-release stages, but generally speaking going forward we should be careful not to change numbers of existing fields because it would break client code that's using the old proto definition. In this case we already have `version`. It's also just reducing unnecessary code changes. Same for some of the other fields (`num_peers`, `num_pairs`, etc).

Reply:
Good point